pagopa/io-web-profile
# [IOPID-1211]: Updating OnetTrust CookieBanner Scripts to ver 11.2023

## Short description

The purpose of this PR is to update the files and scripts of OneTrust for the cookie banner to the latest drafted version.
Please see the jira [task](https://pagopa.atlassian.net/browse/IOPID-1211)

## How to test locally
Insert the following variable into the local environment file:

NEXT_PUBLIC_DEV_MODE=true

Please note that this variable will be omitted when building the production package.
This is necessary to determine if the application is in local development mode or production.
It's required because in production the OT NOTICE ID (OneTrust) it's different

Start the project and ensure that the cookie banner is displayed.
